catalogue_graph/src/transformers/catalogue/raw_work.py

```diff
@@ -16,35 +16,35 @@ class WorkConcept(TypedDict):
 class RawCatalogueWork:
     def __init__(self, raw_work: dict):
         self.raw_work = raw_work
+        self.work_data: dict = self.raw_work.get("data", {})
+        self.work_state: dict = self.raw_work["state"]
 
     @property
     def wellcome_id(self) -> str:
-        wellcome_id = self.raw_work["id"]
-        assert isinstance(wellcome_id, str)
+        wellcome_id: str = self.work_state["canonicalId"]
         return wellcome_id
 
     @property
     def label(self) -> str:
-        label = self.raw_work["title"]
-        assert isinstance(label, str)
+        label: str = self.work_data.get("title", "")
         return label
 
     @property
     def type(self) -> WorkType:
-        concept_type: WorkType = self.raw_work["type"]
-        return concept_type
+        raw_work_type = self.work_data["workType"]
+        work_type: WorkType = "Work" if raw_work_type == "Standard" else raw_work_type
```

Comment:
Perhaps a comment about what this is? I think this denotes if it's a `Series` or a `Standard` work,  but the collisions of the many kinds of "type" possible here makes it a bit confusing.